### ISSUE TITLE
Expose TestGrid API to *.k8s.io domain

### ISSUE THREAD

**Issue by chases2** [kubernetes/k8s.io#4710]
### Type of DNS update:
Create

### Domain being modified:
testgrid-data.k8s.io
testgrid-data.kubernetes.io
testgrid-api.k8s.io
testgrid-api.kubernetes.io

### Existing DNS Record:
Will write in follow-up PR. Target IP is http://34.98.86.92. Hosted with the rest of TestGrid (on GKE)

### Reason for update:
Deploying TestGrid API to K8s community

**Comment by BenTheElder** [comment]
These are live after #4792 
/close
